packages/kit/test/apps/basics/test/test.js

```diff
@@ -1379,3 +1379,24 @@ test.describe.serial('Cookies API', () => {
 		expect(await span.innerText()).toContain('undefined');
 	});
 });
+
+test.describe('Serialization', () => {
+	test('A custom data type can be serialized/deserialized', async ({ page }) => {
+		await page.goto('/serialization-basic');
+		expect(await page.textContent('h1')).toBe('It works!');
+	});
```

Comment:
We need another test that starts somewhere else and then navigates to the page client-side (e.g. you add a link to `/serialization-basic` from some other page, start on said page and make playwright click on that link) in order to test that `_data.json` is also serialized